src/Debugger/Impl/rtvs/R/overrides.R

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in the project root for license information.
+
+view <- function(x, title) {
+  if(is.function(x) || is.data.frame(x) || is.table(x) || 
+     is.matrix(x) || is.list(x) || is.ts(x) || length(x) > 1) {
+    if (missing(title)) {
+      title <- ""
+    }
+    invisible(rtvs:::send_message('View', deparse(substitute(x)), title))
+  } else {
+    print(x)
+  }
+}
+
+open_url <- function(url) {
+  rtvs:::send_message('Browser', url)
```

Comment:
Can you rename the message as well? It's always confusing to me whenever I see it in RHost :)